src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let finalPrice = 0;
```

Comment:
Use `const` for `finalPrice` since its value is not being reassigned, which will ensure the variable is not modified elsewhere in the function.